README.md

```diff
@@ -449,6 +449,36 @@ We use it for the control container because it needs to be available early to gi
 
 Be careful, and make sure you have a similar low-level use case before reaching for host containers.
 
+#### Bootstrap containers settings
+* `settings.bootstrap-containers.<name>.source`: The URI of the container
```

Comment:
Just curious - is this a single container we can run, or can we set up several? The plural on `bootstrap-containers` is confusing me a bit.

Reply:
Ok great. Also, is there support for these containers being hosted in a private registry?

Reply:
Since these host containers, you can use public/private ECR registries, as well as public registries.